### PR TITLE
Group 6 Solution

### PR DESCRIPTION
**Pull request by jsymons**
Jason Symons
Delio Castillo

### CODE REVIEW

**Comment by jangeador** [comment]
Jason Simmons
Delio Castillo

File: tic_tac_toe/game.py [hunk 1]
```diff
@@ -24,7 +26,9 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
+    if isinstance(position, tuple) and len(position) == 2 and position[0] in [0, 1, 2] and position[1] in [0, 1, 2]:
```
Comment (martinzugnoni) [inline tic_tac_toe/game.py]: This line is too long. Pep8 recommends less than 80 characters per line.
Please make it shorter by indenting the `if` statement properly.

**Review by martinzugnoni — APPROVED** [review]
Looks good guys! 🎉 
congrats!

File: tic_tac_toe/game.py [hunk 2]
```diff
@@ -87,18 +121,46 @@ def move(game, player, position):
...
     """
     Returns a string representation of the game board in the current state.
     """
-    pass
+    board_string = '\n'
+    for idx, row in enumerate(game['board']):
+        board_string += '{}  |  {}  |  {}'.format(*row)
+        if idx < 2:
+            board_string += '\n--------------\n'
+    return board_string + '\n'
```
Comment (anshulxyz) [inline tic_tac_toe/game.py:156]: This is nice.

File: tic_tac_toe/game.py [hunk 3]
```diff
@@ -33,7 +38,11 @@ def _board_is_full(board):
 
     :param board: Game board.
     """
-    pass
+    for row in board:
```
Comment (NikolaJankovic) [inline tic_tac_toe/game.py:41]: Great idea formatting it by column and row

**Pull request closed without merging**